### PR TITLE
Implemented "analytics app-versions" command

### PR DESCRIPTION
**Pull request by bryukov**
Implemented the command according to the specification
analytics-app-versions

### CODE REVIEW

File: src/commands/analytics/app-versions.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import * as Path from "path";
+import * as Pfs from "../../util/misc/promisfied-fs";
+import { DefaultApp } from "../../util/profile";
+import * as Os from "os";
+
+const debug = require("debug")("mobile-center-cli:commands:analytics:app-versions");
+
+@help("Shows versions of the application")
+export default class ShowAppVersionsCommand extends AppCommand {
+  @help("Start date")
```
Comment (christav) [inline src/commands/analytics/app-versions.ts]: Include a sample date string in the help so people know the format.

File: src/commands/analytics/app-versions.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  @shortName("s")
+  @longName("start")
+  @hasArg
+  public startDate: string;
+
+  @help("End date")
```
Comment (christav) [inline src/commands/analytics/app-versions.ts]: Include a sample date string so people know the format.

File: src/commands/analytics/app-versions.ts (hunk 1 above)
Comment (bryukov) [inline src/commands/analytics/app-versions.ts]: Done

File: src/commands/analytics/app-versions.ts (hunk 2 above)
Comment (bryukov) [inline src/commands/analytics/app-versions.ts]: Done

**Pull request merged**